frontend/src/pages/Auth/LoginPage.jsx

```diff
@@ -0,0 +1,128 @@
+import { useState, useEffect } from 'react';
+import { Link, useNavigate, useLocation } from 'react-router-dom';
+import AuthService from '../../services/AuthService';
+import { useCookies } from 'react-cookie';
+import { useContext } from 'react';
+import { AuthContext } from '../../utils/AuthContext';
+
+export default function LoginPage() {
+    const [formData, setFormData] = useState({
+        username: '',
+        password: '',
+    });
+    const [error, setError] = useState(null);
+    const [isSubmitting, setIsSubmitting] = useState(false);
+    const navigate = useNavigate();
+    const location = useLocation();
+    const [cookies, setCookie, removeCookie] = useCookies(['refresh_token']);
+    const { login } = useContext(AuthContext);
+
+    useEffect(() => {
+        if (localStorage.getItem('access_token')) {
+            navigate('/');
+        }
+    }, [navigate]);
+
+    const handleChange = (e) => {
+        const { name, value } = e.target;
+        setFormData(prev => ({
+            ...prev,
+            [name]: value
+        }));
+    };
+
+    const handleSubmit = async (e) => {
+        e.preventDefault();
+        setIsSubmitting(true);
+        setError(null);
+
+        try {
+            await login(formData.username, formData.password);
+
+            const userResponse = await AuthService.getCurrentUser();
+            localStorage.setItem('user', JSON.stringify(userResponse.data));
+            
+            const from = location.state?.from?.pathname || '/';
+            navigate(from);
+        } catch (err) {
+            console.error('Full error object:', err);
+            console.error('Error response:', err.response);
```

Comment:
too many errors, better to use some kind
console.error('Login error:', err.response?.data || err.message);